codecov.yaml

```diff
@@ -11,7 +11,8 @@ coverage:
       default:                 
         target: auto
         # don't allow new commits to decrease coverage
-        threshold: 0%
+        # Temporary to merge pr https://github.com/hms-dbmi-cellenics/ui/pull/723
+        threshold: 1%
```

Comment:
I talked with iva aobut this, we'll set the codecov back to 0% after.

The coverage went down because after this we are not testing v1 anymore, which doesn't matter because the next step will be removing all v1 code